src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx

```diff
@@ -351,17 +352,50 @@ export const MapBlockInternal = ({
     );
   }, [dataSetCategoryConfigs]);
 
+  const shouldGroupLocationOptions = dataSetCategories.some(
+    element => element.filter.level === 'localAuthority',
```

Comment:
What about if we have LAD's? They're essentially the same, and we already do this grouping in table tool